hidviz/CMakeLists.txt

```diff
@@ -13,6 +13,9 @@ endif()
 # Find the QtWidgets library
 find_package(Qt5 REQUIRED COMPONENTS Widgets)
 
+find_package(Protobuf REQUIRED)
+include_directories(${PROTOBUF_INCLUDE_DIRS})
```

Comment:
Is this line actually needed? Doesn't `find_package` just handle the includes correctly by itself? If it is indeed needed, would it be possible to add this to `target_include_directories` on line 47?

Reply:
I checked, and on my system it would not compile without explicitly being told where the protobuf include directory is.

I have updated how this is achieved as per your comment.